tests/networking/nethelper/nethelper.go

```diff
@@ -215,3 +274,158 @@ func DefineAndCreateServiceOnCluster(name string, port int32, targetPort int32,
 
 	return err
 }
+
+func DefineAndCreateNadOnCluster(name string, intName string, network string) error {
+	nadOneInteface := nad.DefineNad(name, netparameters.TestNetworkingNameSpace, intName)
+
+	if network != "" {
+		nadOneInteface = nad.RedefineNadWithWhereaboutsIpam(nadOneInteface, network)
+	}
+
+	return globalhelper.APIClient.Create(context.Background(), nadOneInteface)
+}
+
+func GetClusterMultusInterfaces() ([]string, error) {
+	err := defineAndCreatePrivilegedDaemonset()
+	if err != nil {
+		return nil, err
+	}
+
+	podsList, err := globalhelper.GetListOfPodsInNamespace(netparameters.TestNetworkingNameSpace)
+	if err != nil {
+		return nil, err
+	}
+
+	var nodesInterfacesList [][]string
+
+	for _, runningPod := range podsList.Items {
+		nodeIterfaces, err := getInterfacesList(runningPod)
+		if err != nil {
+			return nil, err
+		}
+		nodesInterfacesList = append(nodesInterfacesList, nodeIterfaces)
+	}
+
+	var lastMatch []string
+
+	for _, nodeInterfaces := range nodesInterfacesList {
+		if len(lastMatch) == 0 {
+			lastMatch = findListIntersections(nodesInterfacesList[0], nodeInterfaces)
+		}
+
+		lastMatch = findListIntersections(lastMatch, nodeInterfaces)
+	}
+
+	return lastMatch, nil
+}
+
+func findListIntersections(listA []string, listB []string) []string {
+	var overlap []string
+
+	for _, elementA := range listA {
+		for _, elementB := range listB {
+			if elementA == elementB {
+				overlap = append(overlap, elementA)
+			}
+		}
+	}
+
+	return overlap
+}
+
+func getInterfacesList(runningPod corev1.Pod) ([]string, error) {
+	routes, err := globalhelper.ExecCommand(
+		runningPod,
+		[]string{"ip", "link", "show"},
```

Comment:
why not use the json output and parse it with unmarshal:
ip -j link
{"ifindex":1,"ifname":"lo","flags":["LOOPBACK","UP","LOWER_UP"],"mtu":65536," ....

Just learning about this :-)